Automattic/wp-calypso
# Newsletters: Update style and markup for intro screen

Resolves https://github.com/Automattic/wp-calypso/issues/77309

## Proposed Changes

Update overall appearance and markup of the Newsletter intro screen in onboarding. Changes include: 
   - Changing background image
   - Showing header with Newsletter title
   - Updating title / description / button font
   - Adding button icon
   - Moving Jetpack logo from top to bottom
   
**Before**
<img width="1506" alt="intro-old" src="https://github.com/Automattic/wp-calypso/assets/21228350/b13176bd-a6eb-4d17-8340-568abe512be9">

**After**
<img width="1509" alt="intro-new" src="https://github.com/Automattic/wp-calypso/assets/21228350/9f639753-4220-49d8-987c-67b6fe93f9ec">

## Testing Instructions

Load this branch locally, start a newsletter site at http://calypso.localhost:3000/setup/newsletter/intro, and confirm new design matches the above screenshot.
